src/Management/ResourceBuilder.php

```diff
@@ -270,8 +272,8 @@ private function buildContentTypeField(array $data): Field\FieldInterface
         $hydratorData = [
             'id' => $data['id'],
             'name' => $data['name'],
-            'required' => $data['required'],
-            'localized' => $data['localized'],
+            'required' => $data['required'] ?? null,
+            'localized' => $data['localized'] ?? null,
```

Comment:
Did you actually find a case where they're not set? This was supposed to be fixed in the API.